linux/network_device_ops/apply_new_network_config.yml

```diff
@@ -22,6 +16,28 @@
             ternary('ipv6.method disabled', '')
           }}
 
+    # Some guest OS might automatically add a connection for the new network adapter with
```

Comment:
Is this only needed for static IP assignment or both? since it's already configured with DHCP IP assignment?

Reply:
Both. We don't want the automatically generated the connection file which might still need some modification. So here directly remove it and create a new one as our expected.